.travis.yml

```diff
@@ -13,12 +13,21 @@ install:
 before_script:
 - npm run compile
 script:
-- npm test
+- nvm install 4
```

Comment:
@nikolaylukyanchuk why all these?

Reply:
@karlgroves it check different nodejs versions )
let me fix this )